lint-staged.config.js

```diff
@@ -7,7 +7,7 @@ const isWin = process.platform === 'win32'
 module.exports = {
   '**/*.{js,jsx,ts,tsx}': (filenames) => {
     const escapedFileNames = filenames
-      .map((filename) => `"${isWin ? filename : escape([filename])}"`)
```

Comment:
Were the quotes here causing issues? If i recall correctly without them filenames with spaces or other quirks break without the quotes.

Reply:
It's already handled by lint staged. So the result was `"'path with spaces'"`.
And that caused issues.